### PR TITLE
add config corrected files for userstores

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/learn/configuring-secondary-user-stores.md [hunk 1]
```diff
@@ -7,14 +7,11 @@ and perform operations depending on their roles/permissions. You can
 also configure your own customized user stores and connect them with the
 products as secondary stores.
 
-You can use either the management console to create secondary user
-stores or you can create it manually. These will be stored in the deployment.toml file and use the same format, used to configure
-primary user store.
+You can use either the management console to create secondary user stores or you can create it manually. These will be stored in the deployment.toml file and use the same format, used to configure primary user store.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-secondary-user-stores.md]: shall we correct this to "...and **will** use the same format **that is** used to configure the primary user store."

File: en/docs/learn/configuring-the-realm.md [hunk 2]
```diff
@@ -16,12 +17,8 @@ The following diagram illustrates the required configurations and
 repositories:  
 ![configuring-the-realm.png](../assets/img/using-wso2-identity-server/configuring-the-realm.png)
 
-The following sections include instructions on the above required
-configurations and repositories:
+For more information on Configuring master-datasources.xml, click
+[here](../../administer/configuring-master-datasources.xml).
```
Comment (sherenewso2) [inline en/docs/learn/configuring-the-realm.md]: shouldn't we change this now according to the toml config model?

File: en/docs/learn/configuring-the-system-administrator.md [hunk 3]
```diff
@@ -1,23 +1,21 @@
 # Configuring the System Administrator
 
-User management functionality is provided by default in all WSO2
-Carbon-based products and is configured in the
-`         <PRODUCT_HOME>/repository/conf/user-mgt.xml        ` file.
+User management functionality is provided by default in WSO2 Identity Server and is configured in the
+`         <IS_HOME>/repository/conf/user-mgt.xml        ` file.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-the-system-administrator.md]: should this change according to the toml model?

File: en/docs/learn/configuring-the-realm.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/learn/configuring-the-realm.md]: Need to check if we need these pages in our administer docs.

Comment (gomathyk) [reply en/docs/learn/configuring-the-realm.md]: removed now

**Pull request merged**